website/project/metadata/utils.py

```diff
@@ -0,0 +1,33 @@
+def serialize_meta_schema(meta_schema):
+    if not meta_schema:
+        return None
+    return {
+        'schema_name': meta_schema.name,
+        'schema_version': meta_schema.schema_version,
+        'schema': meta_schema.schema
+    }
+
+def serialize_draft_registration(draft, auth=None):
+    from website.profile.utils import serialize_user  # noqa
+    from website.project.utils import serialize_node  # noqa
+
+    node = draft.branched_from
+
+    return {
+        'pk': draft._id,
+        'branched_from': serialize_node(draft.branched_from, auth),
+        'initiator': serialize_user(draft.initiator, full=True),
+        'registration_metadata': draft.registration_metadata,
+        'registration_schema': serialize_meta_schema(draft.registration_schema),
+        'initiated': str(draft.datetime_initiated),
```

Comment:
Instead of `str(datetime_obj)`, consider using `.isoformat()` to make the code/ format more explicit. (Per docs they should be equivalent) https://docs.python.org/2/library/datetime.html#datetime.date.__str__